src/components/SearchBar.js

```diff
@@ -26,40 +24,45 @@ class SearchBar extends Component {
   findResults = e => {
     e.preventDefault();
     this.props.displayAllSearchResults();
+    this.hideSuggestions();
```

Comment:
I like that you have moved this to the searchBar component. The less we have in app, the better